### PR TITLE
OSDOCS-1145: Reorganized the autoscaling documentation to one file.

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR consolidates the two autoscaling pages into one. This file changes depending on if the user is on ROSA or OSD.

JIRA: https://issues.redhat.com/browse/OSDOCS-1145

PREVIEW:

**ROSA** - https://deploy-preview-38225--osdocs.netlify.app/openshift-rosa/latest/nodes/nodes-about-autoscaling-nodes.html

**OSD** - https://deploy-preview-38225--osdocs.netlify.app/openshift-dedicated/latest/nodes/nodes-about-autoscaling-nodes.html

### CODE REVIEW

File: modules/rosa-adding-node-labels.adoc [hunk 1]
```diff
@@ -10,6 +10,11 @@ Add or edit labels for worker nodes at any time to manage the nodes in a manner
 
 Labels are assigned as key=value pairs. Each key must be unique to the object it is assigned to. Labels do not change or impact the core system values, such as a machine pool ID.
 
+[NOTE]
+====
+Currently, adding node labels on an existing machine pool will only add the labels to new nodes that are created in that pool.  The existing nodes will not have the node label applied.  If you need nodes with the node label, you can either scale down the nodes in the machine pool to zero, which will not work with the 'default' machine pool, add the node label, and then, scale back up to the desired amount.  You can also create a new machine pool and define the node labels at creation time.
```
Comment (0kashi) [inline modules/rosa-adding-node-labels.adoc:15]: This reads funny without the parentheses around the default pool comment.  Can we add them back?

**Comment by 0kashi** [comment]
lgtm

File: _topic_map.yml [hunk 2]
```diff
@@ -203,16 +203,13 @@ Name: Nodes
...
+  File: nodes-machinepools-about
 - Name: Managing worker nodes
   File: rosa-managing-worker-nodes
   Distros: openshift-rosa
-- Name: Enabling autoscaling nodes on a cluster
-  File: nodes-enabling-autoscaling-nodes
-- Name: Disabling autoscaling nodes on a cluster
-  File: nodes-disabling-autoscaling-nodes
-
+- Name: About autoscaling nodes on a cluster
```
Comment (mburke5678) [inline _topic_map.yml:211]: I see that this is an existing term. But, _autoscaling nodes_ threw me off for a moment. I assumed that _autoscaling_ was an adjective; that _autoscaling nodes_ was a type of node. Probably outside the context of this PR.

File: modules/rosa-adding-node-labels.adoc [hunk 3]
```diff
@@ -10,6 +10,11 @@ Add or edit labels for worker nodes at any time to manage the nodes in a manner
 
 Labels are assigned as key=value pairs. Each key must be unique to the object it is assigned to. Labels do not change or impact the core system values, such as a machine pool ID.
 
+[NOTE]
+====
+Currently, adding node labels on an existing machine pool will only add the labels to new nodes that are created in that pool.  The existing nodes will not have the node label applied.  If you need nodes with the node label, you can either scale down the nodes in the machine pool to zero (this will not work with the 'default' machine pool), add the node label, and then, scale back up to the desired amount.  You can also create a new machine pool and define the node labels at creation time.
```
Comment (mburke5678) [inline modules/rosa-adding-node-labels.adoc:15]: s/will only add/adds

You could also move the only closer to _new nodes_: _adds labels to only new nodes_ or _adds labels to new nodes only_.

Comment (mburke5678) [reply modules/rosa-adding-node-labels.adoc:15]: s/and then, / and then ?? (Do you need the comma there?)

File: nodes/nodes-about-autoscaling-nodes.adoc [hunk 4]
```diff
@@ -0,0 +1,78 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+[id="nodes-about-autoscaling-nodes"]
+= About autoscaling nodes on a cluster
+:context: nodes-about-autoscaling-nodes
+toc::[]
+
+ifdef::openshift-dedicated[]
+[IMPORTANT]
+====
+Autoscaling is only available on clusters that were purchased through the Red Hat Marketplace.
```
Comment (mburke5678) [inline nodes/nodes-about-autoscaling-nodes.adoc:10]: s/only available/available only

File: nodes/nodes-about-autoscaling-nodes.adoc [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+[NOTE]
+====
+Only cluster owners and organization admins can scale or delete a cluster.
+====
+
+[id="nodes-enabling-autoscaling-nodes"]
+== Enabling autoscaling nodes on a cluster
+
+Autoscale worker nodes on a cluster to increase or decrease the number of resources available. Autoscaling for worker nodes is configured in the machine pool definition if you are configuring an existing cluster.
```
Comment (mburke5678) [inline nodes/nodes-about-autoscaling-nodes.adoc:30]: s/Autoscale worker nodes/You can enable autoscaling on worker nodes  Or something like that.

File: nodes/nodes-about-autoscaling-nodes.adoc [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+====
+
+[discrete]
+include::modules/rosa-enabling-autoscaling-nodes.adoc[leveloffset=+2]
+endif::[]
+
+[id="nodes-disabling-autoscaling-nodes"]
+== Disabling autoscaling nodes on a cluster
+
+Autoscale worker nodes on a cluster to increase or decrease the number of resources available. Autoscaling for worker nodes is configured in the machine pool definition if you are configuring an existing cluster.
```
Comment (mburke5678) [inline nodes/nodes-about-autoscaling-nodes.adoc:48]: Same [suggestion as above](https://github.com/openshift/openshift-docs/pull/38225/files#r743971328).

File: nodes/nodes-about-autoscaling-nodes.adoc (hunk 5 above)
Comment (mburke5678) [reply nodes/nodes-about-autoscaling-nodes.adoc:30]: _if you are configuring an existing cluster_
What if you are creating a new cluster? Can you set this option? Or, need to deploy the cluster then edit the machine pool definition?

Comment (mburke5678) [reply nodes/nodes-about-autoscaling-nodes.adoc:30]: I wonder if you could combine these two sentences into one? 

_You can enable autoscaling on worker nodes to increase or decrease the number of resources available by editing the machine pool definition for an existing cluster._

Comment (mburke5678) [reply nodes/nodes-about-autoscaling-nodes.adoc:30]: Sorry to pile on. For OCP, the term _resource_ seems to me to apply to CPU and memory. In that perspective, it seems that autoscaling would increase the CPU or memory. _increase or decrease the number of nodes_?

File: nodes/nodes-about-autoscaling-nodes.adoc [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+ifdef::openshift-dedicated[]
+You can disable autoscaling on a cluster using OpenShift Cluster Manager (OCM) console.
+endif::[]
+
+ifdef::openshift-rosa[]
+You can disable autoscaling on a cluster using OpenShift Cluster Manager (OCM) console or the {product-title} CLI.
+
+[NOTE]
+====
+Additionally, you can configure autoscaling on the default machine pool when you xref:../rosa_getting_started/rosa-creating-cluster.adoc#rosa-creating-cluster[create the cluster using interactive mode].
```
Comment (mburke5678) [inline nodes/nodes-about-autoscaling-nodes.adoc:59]: Do you need this note here? You wouldn't need to disable autoscaling on a new cluster, right? And, you aren't interested in enabling autoscaling when looking for info on disabling. (Unless you want to disable it for the individual nodes, then enable it for the cluster.)

**Comment by mburke5678** [comment]
@EricPonvelle The new organization looks great. I had a few thoughts about the text as I was reading it. Feel free to accept/ignore as appropriate. Otherwise LGTM

File: nodes/nodes-about-autoscaling-nodes.adoc [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+include::modules/ocm-enabling-autoscaling-nodes.adoc[leveloffset=+2]
+
+ifdef::openshift-rosa[]
+[NOTE]
+====
+Additionally, you can configure autoscaling on the default machine pool when you xref:../rosa_getting_started/rosa-creating-cluster.adoc#rosa-creating-cluster[create the cluster using interactive mode].
+====
+
+[discrete]
+include::modules/rosa-enabling-autoscaling-nodes.adoc[leveloffset=+2]
```
Comment (mburke5678) [inline nodes/nodes-about-autoscaling-nodes.adoc:42]: In this file, I believe, there is a line: _Decide which machine pool ID you want to configure._
That suggests to me that you can configure autoscaling on only one machine pool ID. If not, maybe make it _machine pool IDs?

If I want to be especially picky, I might say change this to: _Get the ID of the machine pools that you want to confgure._ ;-)

Comment (mburke5678) [reply nodes/nodes-about-autoscaling-nodes.adoc:42]: If I want to be especially picky, I might say change this to: _Get the ID of the machine pools that you want to confgure._

**Pull request merged**